### DISCUSSION TITLE
Question about Configuration needed for a custom token in a comos-sdk based blockchain

### DISCUSSION THREAD

**Discussion by mostafamshk** [cosmos/community#28] [Chain Builders (Cosmos SDK)] [answered]
Hey guys :) 
I have developed a cosmos-SDK-based blockchain with ignite and am now in the **production** phase. 
I have replaced all of the occurrences of `stake` with `n{MY_TOKEN_NAME}` as the [official docs](https://tutorials.cosmos.network/tutorials/9-path-to-prod/4-genesis.html#token-name-s) says.

I don't know if any other configuration is needed for setting my own token in my own custom blockchain application. because the official docs only suggest changing the **staking** unit from `stake` to `n{MY_TOKEN_NAME}`  in the `genesis.json` and I don't know if I have to configure `{MY_TOKEN_NAME}`( NOT `n{MY_TOKEN_NAME}` ) somewhere else in the `genesis.json` or the application. 
I want to start this network on mainnet and I need to be sure I have set this parameter correctly.

**Comment by facundomedica** [comment, marked as answer]
You should check that the genesis file has all the values you consider correct (for example, no "stake" tokens). Also, I strongly believe you should go through one or more testnet phases to make sure your chain works as expected
